### PR TITLE
[EXOD-000] Add beets LP

### PR DESCRIPTION
**Pull request by melondrone**
- Adds **The Monolith LP** bond
- Sets the EXOD-DAI bond to `expired`
  - Doesn't appear in bond list
  - Is still claimable
- Refactor calculation of treasury value on bond page to use treasury value from the graph.
  - Needs to get gOHM price from eth subgraph and gOHM balance from fantom subgraph
Screenshot from 2022-01-11 22-19-36

### CODE REVIEW

**Review by HOVOH — COMMENTED** [review]
Can you add MAI tracking on the treasury graph as well? https://github.com/ExodiaFinance/exodia-subgraph/blob/03ea3503dbec77dbb3d2cab5d696442d2770fc55/schema.graphql#L120.
I also added a gOHM market value which computes balance value + lp value

File: src/slices/AppSlice.ts [hunk 1]
```diff
@@ -114,14 +132,16 @@ export const loadGraphData = createAsyncThunk("app/loadGraphData", async () => {
...
+  const gOhmPrice = parseFloat(ethData ? ethData.data.balances[0].gOhmPrice : 0);
   // const currentBlock = parseFloat(graphData.data._meta.block.number);
 
   return {
     stakingTVL,
     marketCap,
     circSupply,
     totalSupply,
-    treasuryMarketValue,
+    treasuryMarketValue: treasuryMarketValue + treasuryGOhmBalance * gOhmPrice,
```
[the remark this answers is not in the document]
Comment (melondrone) [reply src/slices/AppSlice.ts:144]: Should the `treasuryMarketValue` just contain the entire market value hence no need for computations in the UI?

[the remark this answers is not in the document]
Comment (HOVOH) [reply src/slices/AppSlice.ts]: Yes, it should

**Comment by melondrone** [comment]
@HOVOH 

I've updated the charts as follows with a few lil improvements chucked in: 
- Treasury Chart and Breakdown (pie) includes:
  - EXOD (wsEXOD and EXOD from the monolith)
  - MAI (total mai)
  - DAI (total dai)
  - gOHM (total gOHM)
  - wFTM (total wFTM)
- RVF chart includes: 
  - DAI RFV
  - MAI RFV
- Added 5 day average lines to debt charts (referenced in title, thought it'd be good to have the line too)
- Added growth of supply tooltip to chart (referenced in title, should be in the tooltip too)
Screenshot from 2022-01-12 13-53-12
Screenshot from 2022-01-12 13-53-16
Screenshot from 2022-01-12 14-25-17

**Pull request merged**